### PR TITLE
Feature/credo

### PR DESCRIPTION
**Pull request by yordis**

### CODE REVIEW

File: lib/guardian_db/token.ex [hunk 1]
```diff
@@ -46,7 +48,7 @@ defmodule GuardianDb.Token do
       |> Map.put("claims", claims)
 
     %Token{}
-    |> cast(prepared_claims, [:jti, :typ, :aud, :iss, :sub, :exp, :jwt, :claims])
+    |> cast(prepared_claims, @allowed_fields)
```
Comment (yordis) [inline lib/guardian_db/token.ex:51]: @doomspork this line was `81` 😄 so credo complains about it

Comment (doomspork) [reply lib/guardian_db/token.ex:51]: Sounds good @yordis!  We can change the line length to 120, 80 is out-dated in my opinion 👍

**Comment by yordis** [comment]
@doomspork let me actually add some updates on the `mix.lock`

**Review by doomspork — APPROVED** [review]
@yordis this looks good to me.  Would you like to change the line length to 120 now or handle that later?

File: lib/guardian_db/token.ex (hunk 1 above)
Comment (yordis) [reply lib/guardian_db/token.ex:51]: @doomspork that will require to change `credo` and that is why I dont do it

**Comment by yordis** [comment]
@doomspork I would stick with default `credo` config, at the end it's the same thing anyway

**Comment by doomspork** [comment]
Let me know when you're good to merge @yordis 👍

**Comment by yordis** [comment]
@doomspork ready

**Comment by doomspork** [comment]
Thank you again @yordis 👍

**Pull request merged**